app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -19,6 +19,8 @@ def call
         send_feed_customization_notification unless notification_enqueued
         send_ux_customization_notification unless notification_enqueued
         send_discuss_and_ask_notification unless notification_enqueued
+      rescue ActiveRecord::RecordNotFound => e
+        Rails.logger.error(e)
```

Comment:
Very few people currently have access to production logs so you might want to send this to Datadog instead or Honeybadger would be best if you need to fix something when this occurs.